TDDPracticeTests/TDDPracticeTests/CalculateNumber/CalculatorTests.cs

```diff
@@ -0,0 +1,77 @@
+﻿using FluentAssertions;
+using Xunit.Abstractions;
+
+namespace TestCasesProject.CalculatorCaseStudy
+{
+
+    public class CalculatorTests : IDisposable
+    {
+        private readonly ITestOutputHelper _output;
+        public CalculatorTests(ITestOutputHelper output)
+        {
+            _output = output;
+        }
+
+
+        //dispose all running instances
+        public void Dispose()
+        {
+        }
+
+        [Theory]
+        [MemberData(nameof(TestData))]
+        public void Return_Sum_Of_Numbers(string numbers, int expectedResult)
+        {
+            //Act
+            int result = Calculator.Add(numbers);
+
+            //Assert
+            _output.WriteLine("Return sum of number is executed successfully.");
+            result.Should().NotBe(0).And.BePositive().And.Be(expectedResult, "because they have same values");
+        }
+
+        public static IEnumerable<object[]> TestData => new[]
+        {
+            new object[]{ "1,2", 3 },
+            new object[]{ "7,2", 9 },
+            new object[]{ "1,2,5", 8 },
+            new object[]{ "7,2,9,2", 20 },
+            new object[]{ "7\n,2,9,2", 20 },
+            new object[]{ "//;\n1;2", 3 },
+            new object[]{"//~;\n1~;2~;3", 6}
+        };
+
+        [Theory]
+        [InlineData("1,\n2")]
```

Comment:
why is it a theory is there is only one data set?

Reply:
Removed the theory and converted to fact, because of single input.